modules/reactor/test/tests/meshdivisions/tests

```diff
@@ -0,0 +1,21 @@
+[Tests]
+  issues = '#25901'
+  [mesh_div]
+    design = 'HexagonalGridDivision.md'
+    requirement = 'The system shall be able to divide the mesh using'
+    [single_hex]
+      type = CSVDiff
+      input = 'hexagonal_division.i'
+      csvdiff = 'hexagonal_division_out.csv'
+      detail = 'a hexagonal grid,'
+    []
+    [distributed_hexes]
+      type = CSVDiff
+      input = 'hexagonal_division_positions.i'
+      csvdiff = 'hexagonal_division_positions_out.csv'
+      detail = 'hexagonal grids, centered on user-input positions.'
+      # Steady solve
+      recover = false
```

Comment:
recover generally works with a steady solve

Reply:
umm I dont think so? It just errors in a non-error way and passes the test iirc

Reply:
like this
```
executioners/executioner.test_steady_adapt: Working Directory: /Users/catherinegiudicelli/projects/moose_v5/test/tests/executioners/executioner
executioners/executioner.test_steady_adapt: Running command: /Users/catherinegiudicelli/projects/moose_v5/test/moose_test-devel -i steady-adapt.i --recover --recoversuffix cpr --error --error-override --no-gdb-backtrace
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: *** Info ***
executioners/executioner.test_steady_adapt: Using out_steady_adapt_cp/0004 for recovery.
executioners/executioner.test_steady_adapt: Framework Information:
executioners/executioner.test_steady_adapt: MOOSE Version:           git commit 93a15fdc00 on 2024-01-06
executioners/executioner.test_steady_adapt: LibMesh Version:         
executioners/executioner.test_steady_adapt: PETSc Version:           3.20.1
executioners/executioner.test_steady_adapt: SLEPc Version:           3.20.0
executioners/executioner.test_steady_adapt: Current Time:            Sat Jan  6 15:49:05 2024
executioners/executioner.test_steady_adapt: Executable Timestamp:    Sat Jan  6 09:22:22 2024
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: Parallelism:
executioners/executioner.test_steady_adapt:   Num Processors:          1
executioners/executioner.test_steady_adapt:   Num Threads:             1
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: Mesh: 
executioners/executioner.test_steady_adapt:   Parallel Type:           replicated
executioners/executioner.test_steady_adapt:   Mesh Dimension:          2
executioners/executioner.test_steady_adapt:   Spatial Dimension:       2
executioners/executioner.test_steady_adapt:   Nodes:                   157
executioners/executioner.test_steady_adapt:   Elems:                   132
executioners/executioner.test_steady_adapt:   Num Subdomains:          1
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: Nonlinear System:
executioners/executioner.test_steady_adapt:   Num DOFs:                157
executioners/executioner.test_steady_adapt:   Num Local DOFs:          157
executioners/executioner.test_steady_adapt:   Num Constrained DOFs:    8
executioners/executioner.test_steady_adapt:   Local Constrained DOFs:  8
executioners/executioner.test_steady_adapt:   Variables:               "u" 
executioners/executioner.test_steady_adapt:   Finite Element Types:    "LAGRANGE" 
executioners/executioner.test_steady_adapt:   Approximation Orders:    "FIRST" 
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: Execution Information:
executioners/executioner.test_steady_adapt:   Executioner:             Steady
executioners/executioner.test_steady_adapt:   Solver Mode:             Preconditioned JFNK
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: Cannot recover steady solves!
executioners/executioner.test_steady_adapt: Exiting...
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: WARNING! There are options you set that were not used!
executioners/executioner.test_steady_adapt: WARNING! could be spelling mistake, etc!
executioners/executioner.test_steady_adapt: There are 6 unused database options. They are:
executioners/executioner.test_steady_adapt: Option left: name:--error (no value) source: command line
executioners/executioner.test_steady_adapt: Option left: name:--error-override (no value) source: command line
executioners/executioner.test_steady_adapt: Option left: name:--no-gdb-backtrace (no value) source: command line
executioners/executioner.test_steady_adapt: Option left: name:--recover (no value) source: command line
executioners/executioner.test_steady_adapt: Option left: name:--recoversuffix value: cpr source: command line
executioners/executioner.test_steady_adapt: Option left: name:-i value: steady-adapt.i source: command line
executioners/executioner.test_steady_adapt: Running exodiff: /Users/catherinegiudicelli/projects/moose_v5/framework/contrib/exodiff/exodiff -m  -F 1e-10 -t 5.5e-06  /Users/catherinegiudicelli/projects/moose_v5/test/tests/executioners/executioner/gold/out_steady_adapt.e-s004 /Users/catherinegiudicelli/projects/moose_v5/test/tests/executioners/executioner/out_steady_adapt.e-s004
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt:    *****************************************************************
executioners/executioner.test_steady_adapt:               EXODIFF	(Version: 2.90) Modified: 2018-02-15
executioners/executioner.test_steady_adapt:               Authors:  Richard Drake, rrdrake@sandia.gov           
executioners/executioner.test_steady_adapt:                         Greg Sjaardema, gdsjaar@sandia.gov          
executioners/executioner.test_steady_adapt:               Run on    2024/01/06   15:49:05 CET
executioners/executioner.test_steady_adapt:    *****************************************************************
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: Reading first file ... 
executioners/executioner.test_steady_adapt: Reading second file ... 
executioners/executioner.test_steady_adapt:   FILE 1: /Users/catherinegiudicelli/projects/moose_v5/test/tests/executioners/executioner/gold/out_steady_adapt.e-s004
executioners/executioner.test_steady_adapt:    Title: out_steady_adapt_0003.e
executioners/executioner.test_steady_adapt:           Dim = 2, Blocks = 1, Nodes = 157, Elements = 132, Nodesets = 4, Sidesets = 4
executioners/executioner.test_steady_adapt:           Vars: Global = 1, Nodal = 1, Element = 0, Nodeset = 0, Sideset = 0, Times = 1
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt:   FILE 2: /Users/catherinegiudicelli/projects/moose_v5/test/tests/executioners/executioner/out_steady_adapt.e-s004
executioners/executioner.test_steady_adapt:    Title: out_steady_adapt.e-s004
executioners/executioner.test_steady_adapt:           Dim = 2, Blocks = 1, Nodes = 157, Elements = 132, Nodesets = 4, Sidesets = 4
executioners/executioner.test_steady_adapt:           Vars: Global = 1, Nodal = 1, Element = 0, Nodeset = 0, Sideset = 0, Times = 1
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: exodiff: INFO .. Map option is not needed.
executioners/executioner.test_steady_adapt: Maximum difference between nodal coordinates = 5.90092e-16
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: Nodal coordinates will be compared .. tol:    1e-06 (absolute), floor:        0
executioners/executioner.test_steady_adapt: Time step values will be compared .. tol:  5.5e-06 (relative), floor:    1e-10
executioners/executioner.test_steady_adapt: Global variables to be compared:
executioners/executioner.test_steady_adapt: 	l2_err                           tol:  5.5e-06 (relative), floor:    1e-10
executioners/executioner.test_steady_adapt: Nodal variables to be compared:
executioners/executioner.test_steady_adapt: 	u                                tol:  5.5e-06 (relative), floor:    1e-10
executioners/executioner.test_steady_adapt: No Element variables on either file.
executioners/executioner.test_steady_adapt: No Element Attribute variables on either file.
executioners/executioner.test_steady_adapt: No Nodeset variables on either file.
executioners/executioner.test_steady_adapt: No Sideset variables on either file.
executioners/executioner.test_steady_adapt: Sideset Distribution Factors will be compared .. tol:    1e-06 (relative), floor:        0
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt:   ==============================================================
executioners/executioner.test_steady_adapt:   NOTE: All node and element ids are reported as global ids.
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: Sideset Distribution Factors:
executioners/executioner.test_steady_adapt:   --------- Time step 1, 4.0000000e+00 ~ 4.0000000e+00, rel diff:  0.00000e+00 ---------
executioners/executioner.test_steady_adapt: Global variables:
executioners/executioner.test_steady_adapt: Nodal variables:
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt: exodiff: Files are the same
executioners/executioner.test_steady_adapt: 
executioners/executioner.test_steady_adapt:  
executioners/executioner.test_steady_adapt ...................................................... [recover] OK

```

I dont see the point of running the recover tests on steady execution it just runs the same as steady

Reply:
If you are skipping `recover` testing because it is conceptually a bit silly, then I am ok with that. But if you were actually getting test harness failures with `recover = true`, then that would be concerning because as you point out there's nothing that really happens in part 2

Reply:
the former